xdsl/dialects/func.py

```diff
@@ -86,3 +76,7 @@ class Return(Operation):
     def get(*ops: Union[Operation, SSAValue]) -> Return:
         ops = [op for op in ops] if ops != () else []
         return Return.build(operands=[ops])
+
+
+_operations = [FuncOp, Call, Return]
+_attributes = []
```

Comment:
We need to make clear that these two things here are required. In to class version, this was clearer, IMO.